src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -88,34 +81,38 @@ const convertInclude = async (includeBlock, ...restParams) => {
     return null;
   }
 
-  // This determines if the href has a leading '/'
-  const hrefTypePostion = () => (href.indexOf('/') === 0 ? 1 : 0);
-
-  // This checks if the supportedType is in the correct position of the href
-  const hrefIsSupported = () => supportedType =>
-    href.startsWith(supportedType, hrefTypePostion());
-
-  // This extracts the type from the href
-  const typeExtraction = Object.keys(supportedTypes).find(
-    hrefIsSupported(href),
-  );
+  const { includeType, classification } = includeClassifier({ href, pathname });
 
-  // This determines if the type is supported and returns the include type name
-  const includeType = supportedTypes[typeExtraction];
-  if (!includeType) {
+  if (
+    classification === 'not-supported' ||
+    classification === 'vj-include-not-supporting-amp'
+  ) {
     logger.info(INCLUDE_UNSUPPORTED, {
       type,
+      classification,
       url: href,
     });
     return null;
   }
 
+  let ampSrc;
+  let html;
+  if (classification === 'vj-include-supports-amp') {
+    ampSrc = ampSrcBuilder(href);
+    logger.info(INCLUDE_REQUEST_RECEIVED, {
```

Comment:
```suggestion
    logger.info(INCLUDE_IFRAME_REQUEST_RECEIVED, {
```
Nice addition of the log, you may want to differentiate this log from `INCLUDE_REQUEST_RECEIVED`, that log implies a request is going to be made to the includes endpoint on server, this log says you are building a url to be used in an iframe on the client.